LTCTI_Window.py

```diff
@@ -187,19 +190,29 @@ def LTCTIcallback(self, widget, string):
                 # Now form the string in FOT Request format
                 duration = str(int(days)).zfill(3)+':'+str(int(h)).zfill(2)+":%02d:%02d" % (  m, s)
                 # And then tack it on to the end of the NLET command
-                NLET_cmd += " --LTCTI_duration "+duration+" "            
+                self.RNLE_command_list.extend(['--LTCTI_duration', duration])
     
     
                 # -------------- COMMENT -----------------------------------------
-                # Now append the comment the user made for this event
+                # If the user did not supply a command, create one
                 if user_comment == "":
                     print "USER SUPPLIED NO COMMENT"
                     user_comment = "No Comment"
     
-                NLET_cmd += '--desc "'+user_comment+'" '   
+
+                # Now append the comment  to the argument list
+                self.RNLE_command_list.extend(['--desc', user_comment])
     
             # Now that you have the command line built - execute it
-            os.system(NLET_cmd)
+            try:
+                print('\nRecording the LTCTI event:')
+                x = subprocess.Popen(self.RNLE_command_list)
+                # Now reset the command list to the base command
+                self.RNLE_command_list = [ '/proj/sot/ska3/flight/bin/python', '/home/gregg/GIT_NLET_GUI/nlet_gui/RecordNonLoadEvent.py', 'LTCTI', '--source', 'NLET']
+            except:
```

Comment:
Is a particular error being thrown here? Like say, `IOError`, or something else? We should except that error (or maybe a tuple of the errors you'd expect) so that you don't silently swallow up some unexpected thing. 

Would it ever be useful to capture the error and log to the screen exactly what it was?